lib/LaTeXML/Engine/TeX_Glue.pool.ltxml

```diff
@@ -121,8 +136,13 @@ DefPrimitiveI('\vfilneg', undef, undef);
 # Lastskip
 #----------------------------------------------------------------------
 # \lastskip         iq is 0.0 pt or the last glue or muglue on the current list.
-
-DefRegister('\lastskip' => Glue(0), readonly => 1);
+DefRegister('\lastskip' => Dimension(0), readonly => 1, getter => sub {
+    my $box;
+    for (my $i = $#LaTeXML::LIST ; $i > 0 ; $i--) {
+      my $box = $LaTeXML::LIST[$i];
+      last if !$box || !$box->getProperty('isSkip');
+      return $box->getProperty('width'); }
```

Comment:
Rust's clippy gave me a fun message while porting this loop:
```
error: this loop never actually loops
```

I didn't notice that while reviewing -- may be worth a second look? @brucemiller

Reply:
Maybe the `last` should have been a `next` ?